README.md

```diff
@@ -66,3 +66,5 @@ This section has moved here: https://facebook.github.io/create-react-app/docs/de
 ### `npm run build` fails to minify
 
 This section has moved here: https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify
+
+Contributors: jabramo25, czou1107
```

Comment:
should be alphabetical

Reply:
Actually according to Abramovich et al (2003) it should be by primary contributor.